pkgs/top-level/cuda-packages.nix

```diff
@@ -86,7 +86,7 @@ let
     [
       (import ../development/cuda-modules/setup-hooks/extension.nix)
       (callPackage ../development/cuda-modules/cuda/extension.nix { inherit cudaVersion; })
```

Comment:
We could still add your comment about which `callPackage` this is - this would be the place.